definitions.yaml

```
@@ -86,6 +103,9 @@ definitions:
         description: |
           The [release](https://docs.giantswarm.io/api/#tag/releases) version
           to use in the new cluster
+      availability_zones:
+        description: Number of availability zones a cluster should be spread across.
+        type: number
```

```suggestion
        type: integer
```
This currently generates to a `float64` in `gsclientgen` but should instead be an `int`